drivers/sensor/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ add_subdirectory_ifdef(CONFIG_MHZ19B mhz19b)
 add_subdirectory_ifdef(CONFIG_NCT75 nct75)
 add_subdirectory_ifdef(CONFIG_NTC_THERMISTOR ntc_thermistor)
 add_subdirectory_ifdef(CONFIG_PMS7003 pms7003)
+add_subdirectory_ifdef(CONFIG_INA700 ina700)
```

Comment:
Should be after HP206C

Reply:
I moved this to the `ti` subfolder which I missed in the first round.